### PR TITLE
feat: automatically open two new windows when text file opened

### PR DESCRIPTION
**Pull request by manhatsu**

### CODE REVIEW

File: lua/super-commit/window.lua [hunk 1]
```diff
@@ -7,12 +8,19 @@ function M.set_autocmd()
...
 
+function M.set_user_cmd()
+  vim.api.nvim_create_user_command('OpenVerticalDoubleWindow', function()
+    vert_dbl_window.open()
+  end, {})
+end
+
 function M.set_user_cmd()
   vim.api.nvim_create_user_command('OpenVerticalWindow', function()
     vertical_window.open()
```
[the remark this answers is not in the document]
Comment (asana17) [reply lua/super-commit/window.lua:26]: Or, could you delete all the user commands in a different commit?

We actually don't need these user commands.

File: lua/super-commit/window/vertical_double_window.lua
Comment (asana17) [inline lua/super-commit/window/vertical_double_window.lua:1]: Probably we need to re-think later whether the dedicated files necessary, which depends on its window numbers.

**Review by asana17 — CHANGES_REQUESTED** [review]

File: lua/super-commit/window.lua (hunk 1 above)
[the remark this answers is not in the document]
Comment (manhatsu) [reply lua/super-commit/window.lua]: Deleted all the user commands.

File: lua/super-commit/window/vertical_double_window.lua [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+function M.open()
+  vim.cmd('rightbelow vsplit')
+  vim.cmd('wincmd l')
+  vim.cmd('enew')
+  vim.cmd('wincmd h')
+  vim.cmd('rightbelow split')
+  vim.cmd('wincmd j')
+  vim.cmd('enew')
+  vim.cmd('startinsert')
```
[the remark this answers is not in the document]
Comment (manhatsu) [reply lua/super-commit/window/vertical_double_window.lua]: Deleted that line.

File: lua/super-commit/window/vertical_double_window.lua
Comment (manhatsu) [reply lua/super-commit/window/vertical_double_window.lua:1]: Gathered the functions into `window/open.lua`.

**Review by asana17 — APPROVED** [review]
LGTM.
Could you squash these two commits before merging?

**Pull request merged**